### ISSUE TITLE
Cartridges

### ISSUE THREAD

**Issue by Philsan69** [wavemotion-dave/XEGS-DS#1]
Hi.
First of all, thank you very much for your awesome emulator.
I added it in AtariAge's [Topic for newbies or veterans returning to Atari](https://atariage.com/forums/topic/176545-topic-for-newbies-or-veterans-returning-to-atari/). 

At the moment there are two very good games available in cartridge format only, FloB and Space Harrier.
If not too hard, please add cartridges support.

Regards

**Comment by wavemotion-dave** [comment]
Thanks for the kind comments.  XEGS was the 4th Atari emulator I've worked on for the DS/DSi and I'm proud of the way it turned out. 

Cart support can be added but it comes at a time cost that I wasn't willing to put in given how few games require it.  The three I know about are the two you mentioned plus Sparta DOS-X (not sure if someone would be interested in emulation of DOS but, hey, why not?!).  Given how few games required the use of a .CAR or .ROM, I didn't support it. I think I got burned out on the A5200 .ROM scene where it required a lot of database work to map the cartridge correctly with bank switching and other features... so I just avoided it for XEGS. 

But it's on the list of things to add... someday :)   

I make all my projects open-source on github so hopefully someone with more energy and inclination comes along to add it for me :D
